src/Microsoft.AspNet.Authentication.OpenIdConnectBearer/OpenIdConnectBearerOptions.cs

```diff
@@ -9,19 +9,19 @@
 using Microsoft.IdentityModel.Protocols;
 using Microsoft.IdentityModel.Protocols.OpenIdConnect;
 
-namespace Microsoft.AspNet.Authentication.JwtBearer
+namespace Microsoft.AspNet.Authentication.OpenIdConnectBearer
 {
     /// <summary>
     /// Options class provides information needed to control Bearer Authentication middleware behavior
     /// </summary>
-    public class JwtBearerOptions : AuthenticationOptions
+    public class OpenIdConnectBearerOptions : AuthenticationOptions
     {
         /// <summary>
         /// Creates an instance of bearer authentication options with default values.
         /// </summary>
-        public JwtBearerOptions() : base()
+        public OpenIdConnectBearerOptions() : base()
         {
-            AuthenticationScheme = JwtBearerDefaults.AuthenticationScheme;
+            AuthenticationScheme = OpenIdConnectBearerDefaults.AuthenticationScheme;
```

Comment:
It's misleading that this constant is used for both AuthenticationScheme and Challenge. I'd recommend two different constants, even if they had the same value.